mongodb/leafygreen-ui
# PD-1239: Icons Batch 2

## ✍️ Proposed changes

Adds batch 2 of icons from the joint icon redesign project with the Visual Design team. Also improves the accessibility of our `Link` component.

🎟 _Jira ticket:_ [Add batch 2 icons to LeafyGreen UI](https://jira.mongodb.org/browse/PD-1239)

## 🛠 Types of changes




## ✅ Checklist




## 🧪 How to test changes

For the Icon component:
1. Run Storybook locally
2. Click on the "Icons" nav item and check that all glyphs are adjusted by changing the knobs for size and color.
3. Check other components using the updated icons. Namely the typography and table components.

For the Link component changes:
1. Run Storybook locally
2. Click on the "Typography" nav item.
3. Check that both the open in new tab icon, and the arrow icon have both `role="presentation"` and `aria-hidden="true"`.
